### PR TITLE
changed linter packages, updated dependencies and removed files added…

### PR DESCRIPTION
**Pull request by damfinkel**
… to cookbook

## Summary

- Added new linter packages
- Fixed linter warnings related to parenthesis (prettier changed the default behaviour un v2)
- Removed useRequest, utils, Login and Register whichwere already moved to the cookbook

### CODE REVIEW

File: generators/app/steps/CustomizableSteps/constants.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
 const COMPONENTS_PATH = 'src/components';
 const SCREENS_PATH = 'src/screens';
 const CONFIG_PATH = 'src/config';
-const UTILS_PATH = 'src/utils';
 const SERVICES_PATH = 'src/services';
 const CONSTANTS_PATH = 'src/constants';
 const STYLES_PATH = 'src/scss';
 const DOCS_README_PATH = 'docs';
-const HOOKS_PATH = 'src/hooks';
```
Comment (pabloferro) [inline generators/app/steps/CustomizableSteps/constants.js]: I'd preserve these folders with a README.md. What do you think?

File: generators/app/templates/src/screens/Dashboard/screens/Home/index.tsx [hunk 2]
```diff
@@ -13,24 +9,17 @@ import { reducer, INITIAL_STATE, actionCreators } from './reducer';
...
   useEffect(() => {
     dispatch(actionCreators.setFoo('React'));
   }, [dispatch]);
 
   const handleLogout = () => {
-    userDispatch(authActions.logout());
-    logoutRequest(null);
+    // Perform logout request and on success add the following:
+    // userDispatch(authActions.resetUser());
+    // removeCurrentUserToken();
```
Comment (pabloferro) [inline generators/app/templates/src/screens/Dashboard/screens/Home/index.tsx]: Why did you remove this dispatch?

**Review by pabloferro — CHANGES_REQUESTED** [review]
Looks good! Just two comments.

File: generators/app/templates/src/screens/Dashboard/screens/Home/index.tsx (hunk 2 above)
Comment (pabloferro) [reply generators/app/templates/src/screens/Dashboard/screens/Home/index.tsx]: Also check conflicts

Comment (damfinkel) [reply generators/app/templates/src/screens/Dashboard/screens/Home/index.tsx]: I didn't remove it, I purposedly left them commented because I thought they should be in the logout success callback. But now that I think of it, we should perform these two actions even if logout fails I suppose. So I'll uncomment them

**Review by alejobh — APPROVED** [review]

File: generators/app/templates/src/components/Routes/constants.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+export const ROUTES = [
+  // Leaving this as an example for then the Login screen exists
+  {
+    exact: false,
+    path: PATHS.login,
+    component: () => <span>login</span>,
+    title: 'Routes:loginTitle',
+    description: 'Routes:loginDescription',
+    redirectTo: (user: User | null) => (user ? MAIN_PRIVATE_PATH : undefined)
+  },
```
Comment (pabloferro) [inline generators/app/templates/src/components/Routes/constants.tsx:24]: I don't think we need this example if we have a generator, maybe we can add a comment here about the generator instead.

File: generators/app/templates/src/utils/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+### Global utils files
+
+Add global utils functions and files in this folder. You can check for possible global utils from our [cookbook](https://cookbook.wolox-fearmy.now.sh/recipes/utils)
```
Comment (pabloferro) [inline generators/app/templates/src/utils/README.md:3]: Nice

**Review by pabloferro — APPROVED** [review]
